openshift_images/configuring-samples-operator.adoc

```diff
@@ -45,7 +39,7 @@ endif::openshift-rosa,openshift-dedicated[]
 - .NET
 - Go
 
-* Starting from {product-title} 4.16, Cluster Samples Operator will stop managing non-S2I image streams and templates. You can contact the image stream or template owner for any requirements and future plans. In addition, refer to the link:https://github.com/openshift/library/blob/master/official.yaml[list of the repositories hosting the image stream or templates].
+* The Cluster Samples Operator will stop managing and providing support to the non-S2I samples (image streams and templates). You can contact the image stream or template owner for any requirements and future plans. In addition, refer to the link:https://github.com/openshift/library/blob/master/official.yaml[list of the repositories hosting the image stream or templates].
```

Comment:
Is this correct? We try to avoid parenthesis

```suggestion
* The Cluster Samples Operator will stop managing and providing support to the non-S2I sample image streams and templates. You can contact the image stream or template owner for any requirements and future plans. In addition, refer to the link:https://github.com/openshift/library/blob/master/official.yaml[list of the repositories hosting the image stream or templates].
```

Reply:
@mburke5678 Thanks for the comments. I am retaining the statement as is for technical accuracy.